### PR TITLE
Cleanup running notes on deliveries page

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

**Comment by sofiahag** [comment]
Is this on stage? :-)

File: run_dir/static/js/running_notes.js [hunk 1]
```diff
@@ -203,7 +202,7 @@ function count_cards(){
             }
         }
     });
-    $('.btn_count').append('All <span class="badge bg-secondary">'+all+'</span>');
```
Comment (sofiahag) [inline run_dir/static/js/running_notes.js:206]: Shouldn't the text be at the end? 🤔

**Comment by aanil** [comment]
> Is this on stage? :-)

It is now.

File: run_dir/static/js/running_notes.js (hunk 1 above)
Comment (aanil) [reply run_dir/static/js/running_notes.js:206]: Thats what I changed it to? The `All` is at the end.

Comment (sofiahag) [reply run_dir/static/js/running_notes.js:206]: I know, but should it be? I thought the text is usually positioned after the class? Guess I'm wrong 😆

Comment (aanil) [reply run_dir/static/js/running_notes.js:206]: If you click on Filter all of them have text after the badge. I just changed this one to make it the same as the others.

File: run_dir/design/deliveries.html [hunk 2]
```diff
@@ -96,15 +96,12 @@ <h3>
         <div class="col-10 offset-1">
           <dl class="dl-horizontal hide-incoming">
```
Comment (sofiahag) [inline run_dir/design/deliveries.html:97]: Don't you guys think the running note should have the same width as the FC container? Maybe the "Latest running note" could be above?

Like:

`<dl class="dl-horizontal hide-incoming">`
`<p class="mt-3" style="font-weight:bold;">Latest running note:</p>`

First note as opposed to the second:

Skärmavbild 2023-09-26 kl  14 11 49

**Review by sofiahag — APPROVED** [review]

File: run_dir/static/js/running_notes.js (hunk 1 above)
Comment (sofiahag) [reply run_dir/static/js/running_notes.js:206]: Ah I see what you mean now!

Hmm do we want the button to have the same color as the badge on hover? I probably made the button, I don't remember :-D Or maybe change the "All"- text to white on hover so it's easier to read? 🤔 

Skärmavbild 2023-09-26 kl  14 22 36

**Comment by sofiahag** [comment]
👍🏻

**Pull request merged**